src/Compilation.zig

```diff
@@ -4244,7 +4244,19 @@ fn docsCopyModule(comp: *Compilation, module: *Package.Module, name: []const u8,
             });
         };
         defer file.close();
-        archiver.writeFile(entry.path, file) catch |err| {
+
+        const path = if (std.fs.path.sep == '/') entry.path else blk: {
+            const new_path = try comp.arena.allocSentinel(u8, entry.path.len, 0);
+            @memcpy(new_path, entry.path);
```

Comment:
If the sentinel is needed, then using `dupeZ` would simplify this. However, from what I can tell, there's no need for the sentinel, so just

```zig
const new_path = try comp.arena.dupe(u8, entry.path);
```

would work.